src/ConnectedMode/Binding/ImportBeforeInstallTrigger.cs

```diff
@@ -18,8 +18,10 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
```

Comment:
I have decided to go back to the original implementation and simply trigger it once from within the CTOR.